.travis.yml

```diff
@@ -37,8 +35,9 @@ script:
   - if [[ $PHPCS_TEST ]]; then vendor/bin/phpcs src/ tests/ *.php; fi
   - if [[ $NPM_TEST ]]; then git diff-files --quiet -w --relative=client; fi
   - if [[ $NPM_TEST ]]; then git diff --name-status --relative=client; fi
-  - if [[ $NPM_TEST ]]; then yarn run test; fi
-  - if [[ $NPM_TEST ]]; then yarn run lint; fi
+  - if [[ $NPM_TEST ]]; then yarn coverage; fi
```

Comment:
Does coverage include running the tests?

Reply:
Yep!

Reply:
Good, I feel justified in my approval then :P